src/sentry/api/endpoints/organization_events_has_measurements.py

```diff
@@ -0,0 +1,120 @@
+from datetime import timedelta
+
+import sentry_sdk
+from django.core.cache import cache
+from django.utils import timezone
+from rest_framework import serializers
+from rest_framework.exceptions import ParseError
+from rest_framework.response import Response
+
+from sentry.api.bases import NoProjects, OrganizationEventsV2EndpointBase
+from sentry.snuba import discover
+from sentry.utils.hashlib import md5_text
+from sentry.utils.snuba import is_measurement
+
+MEASUREMENT_TYPES = {
+    "web": [
+        "measurements.fp",
+        "measurements.fcp",
+        "measurements.lcp",
+        "measurements.fid",
+        "measurements.cls",
+    ],
+    "mobile": [
+        "measurements.app_start_cold",
+        "measurements.app_start_warm",
+        "measurements.frames_total",
+        "measurements.frames_slow",
+        "measurements.frames_frozen",
+        "measurements.stall_count",
+        "measurements.stall_stall_total_time",
+        "measurements.stall_stall_longest_time",
+    ],
+}
+
+
+class EventsHasMeasurementsQuerySerializer(serializers.Serializer):
+    transaction = serializers.CharField(max_length=200)
```

Comment:
Would transaction and type be required?

Reply:
Actually should it be name + op?

Reply:
Hm... why op?

Reply:
for frontend measurements, they only exist on pageload ops, and by default we split the performance page between pageload&navigation

Reply:
We ignore the op on the web vitals tab, so even if it has `transaction.op:navigation`, we overwrite it with `transaction.op:pageload` so I don't think this is a problem.